docs/pages/inboxes/group-permissions.mdx

```diff
@@ -327,80 +325,28 @@ try await group.addMembersByInboxId(inboxIds: [inboxId])
 
 :::
 
-### Add members by address
-
-:::code-group
-
-```js [Browser]
-await group.addMembers([walletAddress]);
-```
-
-```js [Node]
-await group.addMembers([walletAddress]);
-```
-
-```tsx [React Native]
-await group.addMembers([walletAddress]);
-```
-
-```kotlin [Kotlin]
-group.addMembers(listOf(walletAddress))
-```
-
-```swift [Swift]
-try await group.addMembers(addresses: [walletAddress])
-```
-
-:::
```

Comment:
We could probably keep this section if you but just put it at the bottom for doing it by identity

`await group.addMembersByIdentity([identity]);`

etc...

Reply:
if we do this we may want to add the newGroup and newDm methods as well that take and Identity 🤷

Reply:
Fantastic - tracking these as fast follows for the docs! 🫡